packages/verification-vue-component/src/components/CertifactionVerification.vue

```diff
@@ -162,15 +162,22 @@ export default {
             const hash = await hashingService.hashFile(item.file)
             let verification = await this.certifactionEthVerifier.verify(hash)
 
-            Vue.set(this.verificationItems, key, { ...item, ...verification })
+            const oldResult = JSON.stringify(this.verificationItems[key])
+            const newResult = JSON.stringify({ ...item, ...verification })
 
-            if (this.offchainVerifier) {
-                verification = await this.offchainVerification(verification)
-            }
+            if (oldResult !== newResult) {
+                Vue.set(this.verificationItems, key, { ...item, ...verification })
 
-            verification.loaded = true
+                if (this.offchainVerifier) {
+                    verification = await this.offchainVerification(verification)
```

Comment:
Shouldn't this offchainVerification done all the time? It could also just change something from the offchainVerification.